packages/graphql/src/translate/translate-read.ts

```diff
@@ -88,8 +88,8 @@ export function translateRead(
         optionsInput.sort = optionsInput.sort?.[node?.singular] || optionsInput.sort;
     }
 
-    if (node.queryOptions) {
-        optionsInput.limit = node.queryOptions.getLimit(optionsInput.limit); // TODO: improve this
+    if (node.limit) {
+        optionsInput.limit = node.limit.getLimit(optionsInput.limit); // TODO: improve this
```

Comment:
apparently, this needs to be improved :thinking: 

Is this comment still relevant?

Reply:
And it gives no context of what needs to be improved... I'll get rid of it. 😛